apps/web/src/services/asset-inventory/collector/collector-main/modules/CollectorItemInfo.vue

```diff
@@ -11,8 +11,10 @@
                             width="1.25rem"
                             height="1.25rem"
                 />
-                <span class="plugin-name">{{ props.item.plugin.name }}</span>
-                <span class="plugin-version">v{{ props.item.plugin.info.version }}</span>
+                <div class="plugin-info">
+                    <span class="plugin-name">{{ props.item.plugin.name }}</span>
+                    <span class="plugin-version">v{{ props.item.plugin.info.version }}</span>
```

Comment:
issue: It seems that `props.item` is sometimes undefined. Would it be better to use a computed property instead?

Reply:
check it again plz!!!